### PR TITLE
Add com.github.tchx84.Flatseal

### PR DESCRIPTION
**Pull request by tchx84**
Flatseal is a graphical utility to review and modify basic permissions from your Flatpak applications.

### CODE REVIEW

**Comment by barthalion** [comment]
bot, build com.github.tchx84.Flatseal

File: com.github.tchx84.Flatseal.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "flatseal",
+            "builddir": true,
```
Comment (barthalion) [inline com.github.tchx84.Flatseal.json]: IIRC it's enabled by default for meson.

Comment (tchx84) [reply com.github.tchx84.Flatseal.json]: It came with Builder's template hah, but I can remove it.

**Comment by tchx84** [comment]
bot, build com.github.tchx84.Flatseal

**Pull request closed without merging**